dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -37,8 +40,29 @@ def self.construct_state_school_id(state_code, row_hash)
     end
   end
 
+  CA_COURSE_CODES = %w(
```

Comment:
Do these map to anything we should comment on, or is the list good enough as is?

Reply:
This is the list of CS course codes that I got from Miranda.